src/Umbraco.Web.UI/umbraco/config/lang/en.xml

```diff
@@ -170,6 +170,7 @@
     <key alias="updateDate">Last edited</key>
     <key alias="updateDateDesc" version="7.0">Date/time this document was edited</key>
     <key alias="uploadClear">Remove file(s)</key>
+    <key alias="url">URL</key>

```

Comment:
To be honest, it would be very good to start at some point the massive cleaning of lang files from the tons of unused 4/6-time's keys... Huge lot of work, I know, but maybe it worths :-)